frontends/web/src/routes/account/send/send.tsx

```diff
@@ -547,14 +542,18 @@ class Send extends Component<Props, State> {
                                     <UTXOs
                                         accountCode={account.code}
                                         active={activeCoinControl}
+                                        explorerURL={account.blockExplorerTxPrefix}
                                         onClose={this.deactivateCoinControl}
                                         onChange={this.onSelectedUTXOsChange}
                                         ref={this.setUTXOsRef}
                                     />
                                 )
                             }
-                            <div className={style.container}>
+                            <div className={`flex flex-row flex-between ${style.container}`}>
                                 <label className="labelXLarge">{t('send.transactionDetails')}</label>
+                                { coinControl && (
+                                    <A href="#" onClick={this.toggleCoinControl} className="labelLarge labelLink">{t('send.toggleCoinControl')}</A>
```

Comment:
Moved down next to transaction details title, but I just noticed that ideally this should be a `<button>`.